net-p2p/qbittorrent/qbittorrent-9999.ebuild

```diff
@@ -4,20 +4,11 @@
 
 EAPI=6
 
-inherit flag-o-matic qmake-utils
+DESCRIPTION="BitTorrent client based on the Qt toolkit and libtorrent-rasterbar"
+HOMEPAGE="http://www.qbittorrent.org"
 
-DESCRIPTION="BitTorrent client in C++ and Qt"
-HOMEPAGE="http://www.qbittorrent.org/"
-
-if [[ ${PV} == *9999 ]]; then
-	inherit git-r3
-	EGIT_REPO_URI="https://github.com/${PN}/qBittorrent.git"
-else
-	MY_P=${P/_}
-	SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.xz"
-	KEYWORDS="~amd64 ~arm ~ppc64 ~x86"
-	S=${WORKDIR}/${MY_P}
-fi
+inherit git-r3
+EGIT_REPO_URI="https://github.com/${PN}/qBittorrent.git"
```

Comment:
Ideally, the 9999 ebuild is kept up to date with each version bump (and possibly in between, if anyone finds the time) and new version bumps are actually created out of 9999.